index.js

```diff
@@ -11,6 +11,12 @@ var defaultOptions = {
     objectName   :   'Css'
 };
 
+var selectors = [];
```

Comment:
Ok, I get it now. Its for supporting multiple files. But its still not a pretty solution... 🤔